### PR TITLE
first draft tooling update

### LINKED ISSUE #864: Listing Epidatr as a tool
I noticed that epidatr isn't listed [here](https://delphi.cmu.edu/tools/); we may want to replace the COVIDCAST R package link with one to epidatr, or add it additionally.

### PR DESCRIPTION
**Pull request by dsweber2**
The tooling is woefully out of date. This is an attempt to fix that. It updates the drop-down menu, and the overview page.

I also added some css to render sub-menu's differently, since the list was rather long.

fix #864

### CODE REVIEW

**Review by carlynvandyke — COMMENTED** [review]
👏 👏 👏  Thank you for doing this!

**Comment by dshemetov** [comment]
Thanks for diving in! Getting this visual bug, maybe we can avoid it by just calling that entry "Dashboards"?

image

File: content/tools/_index.md [hunk 1]
```diff
@@ -3,16 +3,78 @@ title: Tools Overview
 linkTitle: Tools
 layout: single
 ---
+
 ## Forecasting
 
-{{<forecasting>}}
+### [Flu Forecasting (FluSight)](https://www.cdc.gov/flu/weekly/flusight/index.html)
```
Comment (dsweber2) [inline content/tools/_index.md:9]: yeah that's way clearer thanks

**Comment by dsweber2** [comment]
updated based on live feedback during meeting. Could probably use some help on wording and making forecasting wins a bit clearer

**Comment by dshemetov** [comment]
Updated a few things:
- renamed new Packages section to Code, did an editing pass on the language around that
- removed a few unused shortcodes
- changed the Our API link on the landing page to be Our Tools and linked to the Tools Overview instead of the very dense and outdated https://cmu-delphi.github.io/delphi-epidata/
- updated the footer to contain a link to Code

**Review by dshemetov — APPROVED** [review]
I'm pretty happy to merge these changes. The site is not perfect, but certainly a substantial improvement to the out of date information we have live right now.

@melange396 @carlynvandyke let us know what you think and we can make some final tweaks as needed and hopefully merge soon

**Comment by nmdefries** [comment]
Comments from Roni

> The [Delphi Epidata Dashboard](https://deploy-preview-964--cmu-delphi-main.netlify.app/covidcast/) in the pulldown menu is fine, but the [Delphi Epidata Dashboard](https://deploy-preview-964--cmu-delphi-main.netlify.app/tools/covidcast/) link within the ["Dashboards" pulldown item](https://deploy-preview-964--cmu-delphi-main.netlify.app/tools/#dashboards) gets a 404.
>
> Also, in the Tools pulldown menu, "EpiVis" should be replaced with the more descriptive "Epi Signal Visualization".

**Comment by dshemetov** [comment]
@nmdefries Thanks! I'll make a fix in another branch.

**Pull request merged**